src/utils/data.ts

```diff
@@ -27,12 +28,12 @@ export const toParamString = (params, delimiter = ';', transformFn = identity) =
  * @param {Object} params
  * @returns {String}
```

Comment:
now that TypeScript is introduced, the types in JSDoc should be lowercase (e.g., `String` -> `string`)

please do a find+replace for the following:

`{String}` -> `{string}`